Changelogs for Digital Twin Manager UI 1025.0.2 (#4571)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/dtm-1025-1-0-home-dashboard-icon-and-menu-label-changes-now-reflected-in-left-navigation-ctm-2850-2870.md

---

> date: ""

Reviewer: ```suggestion
date: "2026-04-21"
```

> title: "Home dashboard icon and menu label changes now reflected in left navigation. [CTM-2850] (#2870)"

Reviewer 2: ```suggestion
title: "Home dashboard icon and menu label customization now reflected in left navigation"
```

product_area: "Application enablement & solutions"
change_type:
    - value: "change-VSkj2iV9m"
      label: "Fix"
component:
    - value: "component-Tl88RYb4A"
      label: "Digital Twin Manager"
build_artifact:
    - value: "tc-wYIY0MBDO"
      label: "dtm"
ticket: "CTM-2850"
version: "1025.1.0"
---

> In Digital Twin Manager, the Icon and Menu label settings in the

Reviewer 2: ```suggestion
In Digital Twin Manager (DTM), the icon and menu label settings in the
```

> dashboard configuration were being overridden by the DTM app

Reviewer 2: ```suggestion
dashboard configuration were being overridden by the DTM application
```

configuration, preventing user customizations from appearing in the left

> navigation. Previously, when user changed the Icon or Menu label in the

Reviewer 2: ```suggestion
navigation. Previously, when users changed the icon or menu label in the
```
